service/telemetry/factory.go

```diff
@@ -68,11 +69,11 @@ func NewFactory() Factory {
 	return newFactory(createDefaultConfig,
 		withLogger(func(ctx context.Context, set Settings, cfg component.Config) (*zap.Logger, log.LoggerProvider, error) {
 			c := *cfg.(*Config)
-			return newLogger(ctx, set, c)
+			return newLogger(set, c)
 		}),
-		withTracerProvider(func(ctx context.Context, set Settings, cfg component.Config) (trace.TracerProvider, error) {
+		withTracerProvider(func(_ context.Context, set Settings, cfg component.Config) (trace.TracerProvider, error) {
```

Comment:
can the `ctx` in the `withLogger` function also be `_`?

Reply:
yup, this has been fixed